src/Orleans.sln

```diff
@@ -74,6 +74,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Nuget", "Nuget", "{014CD19F
 		NuGet\Microsoft.Orleans.OrleansManager.nuspec = NuGet\Microsoft.Orleans.OrleansManager.nuspec
 		NuGet\Microsoft.Orleans.OrleansProviders.nuspec = NuGet\Microsoft.Orleans.OrleansProviders.nuspec
 		NuGet\Microsoft.Orleans.OrleansRuntime.nuspec = NuGet\Microsoft.Orleans.OrleansRuntime.nuspec
+		..\..\..\..\Users\Gutemberg\Desktop\OrleansTelemetryConsumers.NewRelic\Microsoft.Orleans.OrleansTelemetryConsumers.NewRelic.nuspec = ..\..\..\..\Users\Gutemberg\Desktop\OrleansTelemetryConsumers.NewRelic\Microsoft.Orleans.OrleansTelemetryConsumers.NewRelic.nuspec
```

Comment:
Shouldn't this point to the copy in the Orleans source tree, rather than in your user account?